src/plugins/data/server/index_patterns/routes/runtime_fields/create_runtime_field.ts

```diff
@@ -53,19 +54,18 @@ export const registerCreateRuntimeFieldRoute = (
         throw new Error(`Field [name = ${name}] already exists.`);
       }
 
-      indexPattern.addRuntimeField(name, runtimeField);
-
-      const addedField = indexPattern.fields.getByName(name);
-      if (!addedField) throw new Error(`Could not create a field [name = ${name}].`);
```

Comment:
There is no need to add this validation as adding a field will either throw (in which case the error will be displayed from that throw) or succeed.